core/src/forest/ForestOptions.cpp

```diff
@@ -62,6 +62,31 @@ ForestOptions::ForestOptions(uint num_trees,
   }
 }
 
+ForestOptions::ForestOptions(uint num_trees,
+                             size_t ci_group_size,
+                             double sample_fraction,
+                             uint mtry,
+                             uint min_node_size,
+                             bool honesty,
+                             double honesty_fraction,
+                             bool honesty_prune_leaves,
+                             double alpha,
+                             double imbalance_penalty,
+                             uint num_threads,
+                             uint random_seed,
+                             const std::vector<size_t>& sample_clusters,
+                             uint samples_per_cluster,
+                             uint blocklength) :
+    ci_group_size(ci_group_size),
```

Comment:
you do not need these guys, since they are included in the delegated constructor.

Reply:
and compiler does not allow these guys to be used along with a delegated constructor

Reply:
It doesn't work if I remove all the arguments in the constructor. Can you write a sample for this one? since our case is different from the "Foo" example.